README.md

```diff
@@ -49,10 +49,10 @@ We use [pylint](https://pylint.org) as Python linter.
 
 ## Coding Style
 
-In general, we follow PEP8 Python coding style and Google pyguide
-However, if PEP8 and Google pyguide has conflict style, PEP8 take place.
+In general, we follow [PEP 8](https://www.python.org/dev/peps/pep-0008/) Python coding style and Google pyguide
```

Comment:
Should we add a `.` at the end?
`In general, we follow [PEP 8](https://www.python.org/dev/peps/pep-0008/) Python coding style and Google pyguide.`

Reply:
Good point! Resolved.